playbooks/roles/nginx/defaults/main.yml

```diff
@@ -12,6 +12,7 @@ NGINX_EDXAPP_CUSTOM_REDIRECTS: {}
 NGINX_ELASTICSEARCH_HOST: localhost
 
 NGINX_ENABLE_SSL: False
+NGINX_KIBANA_ENABLE_SSL: False
```

Comment:
This should actually be called `KIBANA_ENABLE_SSL` and it should be included in the kibana role defaults.